src/test/rgw/rgw_multi/tests.py

```diff
@@ -586,6 +586,42 @@ def test_bucket_create():
     for zone in zonegroup_conns.zones:
         assert check_all_buckets_exist(zone, buckets)
 
+def test_bucket_create_with_tenant():
+    zonegroup = realm.master_zonegroup()
+    zonegroup_conns = ZonegroupConns(zonegroup)
+    primary = zonegroup_conns.rw_zones[0]
+    secondary = zonegroup_conns.rw_zones[1]
+
+    access_key = 'abcd'
+    secret_key = 'efgh'
+    tenant = 'testx'
+    uid = 'test'
+
+    cmd = ['user', 'create', '--tenant', tenant, '--uid', uid, '--access-key', access_key, '--secret-key', secret_key, '--display-name', 'tenanted-user']
+    primary.zone.cluster.admin(cmd)
```

Comment:
please cover the rest of the function with try/finally to clean up this user, ex:
```py
    cmd = ['user', 'create', '--tenant', tenant, '--uid', uid, '--access-key', access_key, '--secret-key', secret_key, '--display-name', 'tenanted-user']
    primary.zone.cluster.admin(cmd)
    try:
        ...
    finally:
        cmd = ['user', 'rm', '--tenant', tenant, '--uid', uid, '--purge-data']
        primary.zone.cluster.admin(cmd)
```
if we don't do this cleanup, we won't be able to rerun the test against the same cluster

Reply:
sorry, updated. thanks!